maxim/logger/livekit/gemini/gemini_realtime_session.py

```
@@ -0,0 +1,266 @@
+import functools
```

Pylint (3.3.7)

[convention] Missing module docstring

(C0114)